OrbitQt/orbitmainwindow.cpp

```diff
@@ -705,7 +708,30 @@ void OrbitMainWindow::on_actionHelp_triggered() { GOrbitApp->ToggleDrawHelp(); }
 
 void OrbitMainWindow::on_actionFeedback_triggered() { ShowFeedbackDialog(); }
 
+void OrbitMainWindow::ShowCaptureOnSaveWarningIfNeeded() {
+  const QString skip_capture_warning =
+      QString::fromStdString("SkipCaptureVersionWarning");
+  if (settings.value(skip_capture_warning, 0).toInt() == 0) {
+    QMessageBox message_box;
+    message_box.setText(
+        "Note: Old captures might be unavailable in the future versions. "
```

Comment:
Suggested wording:
"Note: Captures saved with this version of Orbit might be incompatible with future versions. Please check release notes for more information"

This makes it clear that captures saved with the current version might become incompatible. Also, fix wording, "unavailable" doesn't really sound right to me.